_data/isupport.yaml

```diff
@@ -452,6 +528,31 @@ values:
             - "SILENCE=15"
 
     -
+        name: SSL
```

Comment:
I believe this token was removed from Insp as per https://github.com/inspircd/inspircd/commit/232061 , I'd mark it as obsolete. Missing letter in "foun on" just below in the comment.